deployments/ansible/roles/splunk-otel-collector/handlers/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+# handlers file for splunk-otel-collector
+
+- name: Reload systemd daemon
+  ansible.builtin.systemd:
+    daemon_reload: yes
+  listen:
+    - reload otel-collector
```

Comment:
nit `restart splunk-otel-collector` and `restart td-agent` seem more accurate?

Reply:
this includes systemd reload, that's why I called it reload. restart is good as well

Reply:
renamed to restart